scripts/enforce_git_lfs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+output=$(git checkout .)
+
+if echo "$output" | grep -q "files that should have been pointers, but weren't"; then
```

Comment:
You can do a substring match natively in bash with https://stackoverflow.com/a/229606/132047 if you want